crates/uv-client/src/base_client.rs

```diff
@@ -139,9 +139,9 @@ impl<'a> BaseClientBuilder<'a> {
         let client = self.client.clone().unwrap_or_else(|| {
             if CryptoProvider::get_default().is_none() {
                 // Use aws_lc_rs as the default TLS provider to support more SSL cert algorithms
-                aws_lc_rs::default_provider()
-                    .install_default()
-                    .expect("failed to install aws_lc_rs as default TLS provider");
+                if aws_lc_rs::default_provider().install_default().is_err() {
+                    warn_user_once!("Failed to install aws_lc_rs as the default TLS provider.");
+                }
```

Comment:
It looks like the error attached to the result is just `Self` and has no information, right?

Reply:
Yes I understand the error holds the `CryptoProvider` that was previously installed as the default in that case.

Reply:
I guess we could capture it to say what we're using instead but it doesn't seem critical.

Reply:
Ah yeah I thought about doing that but I wasn't sure the `CryptoProvider` had a nice human-readable name field we could use: https://docs.rs/rustls/latest/rustls/crypto/struct.CryptoProvider.html